lib/ibandit/iban_assembler.rb

```diff
@@ -39,10 +46,7 @@ def self.assemble_at_bban(opts)
       #
       # Padding:
       #   Add leading zeros to account number if < 11 digits.
```

Comment:
These comments (especially padding) are more relevant to the `LocalDetailsCleaner`. Move them in a separate PR?